nasa/ow_autonomy
# Enforce pan/tilt limits, and related improvements.

## Summary

- **IMPORTANT**: This branch must be used with the same branch in `ow_simulator`. 
- PLEXIL now enforces pan range of [-3.2, 3.2] and tilt range of (-pi/2, pi/2).
- Angles are compared with regard for equivalences.
- Tolerances now match those used in simulator.
- Improved pan/tilt test plan.

## Test

1. Start any simulator world.
2. Adjust Gazebo to see the antenna clearly.
3. Run the plan TestAntennaMovement.   E.g.
```
$ roslaunch ow_plexil ow_exec.launch plan:=TestAntennaMovement.plx
```
All 13 tests should succeed.  They command a wide range of pans and tilts.
